tests/acceptance/ape-config.yaml

```diff
@@ -5,7 +5,7 @@ plugins:
 
 dependencies:
   - name: nucypher-contracts
-    github: nucypher/nucypher-contracts
+    github: kprasch/nucypher-contracts
```

Comment:
Just putting a note here to change this before we merge.

Reply:
```suggestion
    github: nucypher/nucypher-contracts
```